pkg/generators/snippets/enum.go

```diff
@@ -23,8 +23,10 @@ func GenerateEnumSetter(inputType *types.Type, enumOptions []string) (string, ge
 	raw := ""
 
 	for _, val := range enumOptions {
-		raw += fmt.Sprintf(`const $.name$%s $.name$ = "%s"`, toSuffix(val), val)
-		raw += "\n"
+		if val != "" {
+			raw += fmt.Sprintf(`const $.name$%s $.name$ = "%s"`, toSuffix(val), val)
+			raw += "\n"
+		}
```

Comment:
I'm consued on how an empty `ref=` would cause this to break.

`GenerateEnumSetter` gets called [here](https://github.com/kanopy-platform/code-generator/blob/main/pkg/generators/builder/builder.go#L122)

GetEnumOptions -> [ExtractArg](https://github.com/kanopy-platform/code-generator/blob/main/pkg/generators/tags/tags.go#L57) should only return something if `enum` is specified. The example in the description only has an empty `ref=`. So I wouldn't think it would return anything.

So then here the `GenerateEnumSetter` input should be an empty string slice, and this section of code wouldn't run?

Reply:
Ii'll update my description.  Its not an empty ref, its an empty enum.